servicetalk-examples/grpc/protoc-options/build.gradle

```diff
@@ -74,6 +74,7 @@ protobuf {
           outputSubDir = "java"
           // Option to append a suffix to type names to avoid naming collisions with other code generation.
           option 'typeNameSuffix=St'
+          option 'defaultServiceMethods=true'
```

Comment:
My expectation was that we can add this behavior without introducing a new property, but if that caused a change in `BlockingProtocOptionsServer` (services with a single RPC that were used as lambda) then it's good to protect by an option.

Please add a comment above clarifying what this option does.

Reply:
Yeah, this was put behind an option because it would break code that treated these interfaces as functional, in cases where it was able to.